README.md

```diff
@@ -187,7 +189,7 @@ The name of the application pool for the application.
 
 #### `virtual_directory`
 
-The IIS Virtual Directory to convert to an application on create.
+The IIS Virtual Directory to convert to an application on create. Similar to iis_application, iis_virtual_directory uses composite namevars.
```

Comment:
Doc updates should not be part of release prep

Reply:
Working on getting rid of this.